### PR TITLE
Initial CloudAMQP terraform setup

### PR DESCRIPTION
**Pull request by jahpola**

### CODE REVIEW

File: cloudamqp/main.tf [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Create a new cloudamqp instance
+resource "cloudamqp_instance" "instance" {
+  name        = "dev-mqtt"
+  plan        = "squirrel-1"
+  region      = "google-compute-engine::${var.region}"
+  tags        = ["dev"]
+  rmq_version = "3.10.8"
```
Comment (haphut) [inline cloudamqp/main.tf:7]: Are software versions usually specified in `.tf` or `.tfvars` files? I guess we won't find a Dependabot-like for either.

Comment (jahpola) [reply cloudamqp/main.tf:7]: dependabot won't work with Terraform code. Each terraform provider or terraform version has to be manually checked. Also in this case changing rmq_version is very risky via automation. Have to test the terraform logic here to see how it works (does it create a new and replace or just update the version)

File: cloudamqp/main.tf [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+resource "cloudamqp_notification" "recipient_01" {
+  instance_id = cloudamqp_instance.instance.id
+  type        = "email"
+  value       = var.notification_email
+  name        = "Default"
+}
+
+resource "cloudamqp_custom_domain" "settings" {
+  instance_id = cloudamqp_instance.instance.id
+  hostname    = "dev.mqtt.apc.lmj.fi"
```
[the remark this answers is not in the document]
Comment (haphut) [reply cloudamqp/main.tf:20]: Sounds good.

Regarding `terraform import`, to me this PR looks like a clean start already, though, apart from the version numbers and the hostname.

[the remark this answers is not in the document]
Comment (jahpola) [reply cloudamqp/main.tf:20]: I used terraform import as a starting point for seeing all needed variables. I will change this to clean start via sandbox name.

**Pull request merged**